js/constants/config.js

```diff
@@ -40,5 +40,6 @@ module.exports = {
     authUrl: (userId) => `${vaultHost}/v1/users/${userId}`,
     replacementUrl: adHost
   },
-  braveExtensionId: 'mnojpmjdmbbfmejpflffifhffcmidifd'
+  braveExtensionId: 'mnojpmjdmbbfmejpflffifhffcmidifd',
+  PDFJSExtensionId: 'ckglimbbooghmggmkikhbolcfmgniemo'
```

Comment:
Looks like this is all worked out in the follow up commit by the way by Yan. I got the ID from the one that is used when opening PDFs from a link.